web/templates/compose_control_buttons.hbs

```diff
@@ -24,11 +24,6 @@
     <div class="compose_control_button_container preview_mode_disabled" data-tooltip-template-id="add-global-time-tooltip" data-tippy-maxWidth="none">
         <a role="button" class="compose_control_button zulip-icon zulip-icon-time time_pick" aria-label="{{t 'Add global time' }}" tabindex=0></a>
     </div>
-    {{#unless message_id}}
```

Comment:
The moving of this button to it's own section at the end should be in its own prep commit before the main one adding the todo modal button